hnn_core/tests/test_params.py

```diff
@@ -8,9 +8,7 @@
 
 import pytest
 
-from hnn_core import (read_params, Params, jones_2009_model, convert_to_hdf5,
-                      Network)
-from hnn_core.hnn_io import read_network
+from hnn_core import read_params, Params, jones_2009_model, convert_to_hdf5
```

Comment:
```suggestion
from hnn_core import read_params, Params, jones_2009_model
```
?

Reply:
This actually is updated in the subsequent PR that changes this function to convert_to_json.